templates/tools/dockerfile/test/ruby_debian11_arm64/Dockerfile.inja

```diff
@@ -12,12 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM arm64v8/debian:11
+FROM arm64v8/ruby:3.1-bullseye
```

Comment:
@apolcyn Was your intent to leave this one using a direct Ruby image for now? :-)

Reply:
yes, and if this works out I'm thinking to move other images as well :)